homeyess/website/views.py

```diff
@@ -396,4 +418,9 @@ def confirm_ride(request, ride_id):
     ride.start_datetime, ride.pickup_datetime, ride.end_datetime, _ = getTimes(td_vec, ride.interview_datetime)
     ride.save()
 
+    if volunteer_profile != None:
+        send_message("Ride to {} has been confirmed".format(ride.interview_address), volunteer_profile)
+    if homeless_profile != None:
+        send_message("Ride to {} has been confirmed".format(ride.interview_address), homeless_profile)
```

Comment:
They might not know the terminology `confrimed`. Can you say something like `your ride has been matched with a volunteer`